src/components/UsersList/UsersList.tsx

```diff
@@ -96,35 +136,116 @@ export function UsersList({
-          );
-        })}
-      </GridList>
+      {removedUsername && (
+        <Modal
+          close={closeModal}
+          modalId="removeUserSuccess"
+          headingId={modalHeadingId(removedUsername)}
+        >
+          <Alert type="success" id={modalHeadingId(removedUsername)}>
+            <strong>{removedUsername}</strong> has successfully been removed.
+          </Alert>
+        </Modal>
+      )}
+
+      <Table
+        caption="users for this organization"
+        sortText="This table is now sorted by Account Name in descending order."
+      >
+        <TableHead>
+          <TableHeadCell
+            data="account name"
+            sortDir={sortParam === 'username' ? sortDir : 'unsorted'}
+            onSortClick={onSortAction}
+          />
+          <TableHeadCell
+            data="organization roles"
+            sortDir={sortParam === 'orgRolesCount' ? sortDir : 'unsorted'}
+            onSortClick={onSortAction}
+          />
+          <TableHeadCell
+            data="access permissions"
+            sortDir={sortParam === 'spaceRolesCount' ? sortDir : 'unsorted'}
+            onSortClick={onSortAction}
+          />
+          <TableHeadCell
+            data="expires"
+            sortDir={sortParam === 'daysToExpiration' ? sortDir : 'unsorted'}
+            onSortClick={onSortAction}
+          />
+          <TableHeadCell
+            data="last login"
+            sortDir={sortParam === 'lastLogonTime' ? sortDir : 'unsorted'}
+            onSortClick={onSortAction}
+          />
+          <TableHeadCell />
+        </TableHead>
+
+        <TableBody>
+          {currentUsers.map((user, index) => (
+            <TableRow key={`table-row-${index}`}>
+              <TableCell
+                colName="account name"
+                rowheader={true}
+                sort={sortParam === 'username'}
+              >
+                <div className="display-flex flex-justify">
+                  <span className="mobile-lg:text-bold maxw-card-lg text-ellipsis">
+                    <Username
+                      user={user}
+                      serviceAccount={serviceAccounts[user.username]}
+                    />
+                  </span>
+                </div>
+              </TableCell>
+
+              <TableCell
+                colName="organization roles"
+                sort={sortParam === 'orgRolesCount'}
+              >
+                <UsersListOrgRoles
+                  href={`/orgs/${orgGuid}/users/${user.guid}/org-roles`}
+                  orgRolesCount={user.orgRolesCount}
+                />
+              </TableCell>
+
+              <TableCell
+                colName="access permissions"
+                sort={sortParam === 'spaceRolesCount'}
+              >
+                <UsersListSpaceRoles
+                  href={`/orgs/${orgGuid}/users/${user.guid}`}
+                  spaceRolesCount={user.spaceRolesCount}
+                  spacesCount={Object.keys(spaces).length}
```

Comment:
[non-blocking suggestion] If we wanted to cut out a little bit of extra counting, this could be calculated before the loop starts and just passed in, rather than calculated on each loop.

Reply:
good point—fixed!